app/uk/gov/hmrc/customs/declaration/controllers/FileUploadController.scala

```diff
@@ -16,10 +16,14 @@
 
 package uk.gov.hmrc.customs.declaration.controllers
 
+import java.util.UUID
+
 import javax.inject.{Inject, Singleton}
```

Comment:
multiple unused imports - are you creating a local squashed commit branch and going through all intellij warnings before committing the branch?